bindings/matrix-sdk-ffi/src/notification_settings.rs

```diff
@@ -319,6 +319,16 @@ impl NotificationSettings {
         }
     }
 
+    /// Check whether [MSC 4028 push rule][rule] is enabled on the homeserver.
+    ///
+    /// [rule]: https://github.com/matrix-org/matrix-spec-proposals/blob/giomfo/push_encrypted_events/proposals/4028-push-all-encrypted-events-except-for-muted-rooms.md
+    pub async fn can_homeserver_push_encrypted_event_to_device(&self) -> bool {
```

Comment:
There's a bit of logic here, and I think that this could be useful for other clients. Could this method go to the SDK, next to `unstable_features`, please? (and add a test for it too :pray:)

Reply:
of course! good hint. let me know if what I pushed goes in the direction of ur suggestion 👍